ambari-web/app/views/common/modal_popup.js

```diff
@@ -36,7 +36,7 @@ App.ModalPopup = Ember.View.extend({
   disableSecondary: false,
   disableThird: false,
   primaryClass: 'btn-success',
-  secondaryClass: 'btn-default',
```

Comment:
By design secondary button should have "btn-default" class

Reply:
Why? It seems like a secondary button should have the btn-secondary class. Otherwise, why is that part of the spec?

Reply:
Bootstrap buttons don't have class "btn-secondary" if you change class secondary button will have broken styling.

Reply:
Yeah, we have "btn-secondary" class, change is good.